content/storage-history/storage.textile

```diff
@@ -0,0 +1,109 @@
+    </a>
+
+    Scroll down and click "Add new rule":
+
+    <a href="/images/screenshots/storage/dashboard-channel-rule.png" target="_blank">
+        <img src="/images/screenshots/storage/dashboard-channel-rule.png" style="width: 100%" alt="Add new rule button">
+    </a>
+
+    Tick "Persist all messages":
+
+    <a href="/images/screenshots/storage/dashboard-persist-all.png" target="_blank">
+        <img src="/images/screenshots/storage/dashboard-persist-all.png" style="width: 100%" alt="Persist all messages checkbox">
+    </a>
+
+    Click "Create channel rule":
+
+    <a href="/images/screenshots/storage/dashboard-save.png" target="_blank">
+        <img src="/images/screenshots/storage/dashboard-save.png" style="width: 100%" alt="Create channel rule button">
+    </a>
+
+
+h2(#persist-last-message). Persist last message - 365 days
+
+You can also choose to persist just "the last message":/general/channel-rules-namespaces on a channel, which persists the last published message for one year. 
+
+This message is not retrieved using the History API, you must use the "channel rewind":/channels/options/rewind mechanism, attached to a channel with @rewind=1@. Note that only regular channel messages can be stored, not presence messages. 
+
+This option for persistence is illustrated in the following diagram:
+
+<a href="/images/diagrams/history-persist-last-message.png" target="_blank">
+  <img src="/images/diagrams/history-persist-last-message.png" style="width: 100%" alt="Persist Last Message">
+</a>
+
+Foldable box
+
+    Select the settings tab from your Ably "dashboard":https://ably.com/dashboard :
+
+    <a href="/images/screenshots/storage/dashboard-settings.png" target="_blank">
+        <img src="/images/screenshots/storage/dashboard-settings.png" style="width: 100%" alt="Settings button">
+    </a>
+
+    Scroll down and click "Add new rule":
+
+    <a href="/images/screenshots/storage/dashboard-channel-rule.png" target="_blank">
+        <img src="/images/screenshots/storage/dashboard-channel-rule.png" style="width: 100%" alt="Add new rule button">
+    </a>
+
+    Tick "Persist last message":
+
+    <a href="/images/screenshots/storage/dashboard-persist-last.png" target="_blank">
+        <img src="/images/screenshots/storage/dashboard-persist-last.png" style="width: 100%" alt="Persist last message checkbox">
+    </a>
+
+    Click "Create channel rule":
+    
+    <a href="/images/screenshots/storage/dashboard-save.png" target="_blank">
+        <img src="/images/screenshots/storage/dashboard-save.png" style="width: 100%" alt="Create channel rule button">
+    </a>
+
+
+h2(#outside-persistence). Store messages outside of Ably
+
+The final option for persisting messages is to use integrations to store your messages outside of Ably. With integrations you can set up external webhooks so messages sent to a channel notify HTTP endpoints, serverless functions or other services for each event. 
```

Comment:
This paragraph is a little confusing. I think we need to mention that integrations are triggered based on integration rules. It's also worth avoiding using terminology like "the final option" as that implies someone must read the entire page for context.